modules/v2x/v2x_proxy/proto_adapter/proto_adapter.h

```diff
@@ -0,0 +1,59 @@
+/******************************************************************************
+ * Copyright 2020 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#pragma once
+
+#include <memory>
+
+#include "modules/v2x/proto/v2x_carstatus.pb.h"
+#include "modules/v2x/proto/v2x_junction.pb.h"
+#include "modules/v2x/proto/v2x_obu_rsi.pb.h"
+#include "modules/v2x/proto/v2x_obu_traffic_light.pb.h"
+#include "modules/v2x/proto/v2x_rsi.pb.h"
+#include "modules/v2x/proto/v2x_traffic_light.pb.h"
+
+#include "modules/map/hdmap/hdmap.h"
+#include "modules/map/hdmap/hdmap_common.h"
+
+namespace v2x {
+namespace proxy {
+typedef ::apollo::v2x::SingleTrafficLight_Color OSLightColorE;
+typedef ::apollo::obu::v2x::SingleTrafficLight_Color OBULightColorE;
```

Comment:
Avoid `typedef`s. And Prefer not using `using` in header files.

Reply:
The names of these types are too long, so use _typedef_ to shorten the names, and these definitions are in the namespace, so there will be no symbol conflicts. If these _typedef_ s are deleted, a lot of meaningless repetitive code will be added.

Reply:
> The names of these types are too long, so use _typedef_ to shorten the names, and these definitions are in the namespace, so there will be no symbol conflicts. If these _typedef_ s are deleted, a lot of meaningless repetitive code will be added.

Good question. Then I would suggest using `using` rather than `typedef`. Again, using `E` to mean enumeration doesn't confirm to Apollo coding style.